ckanext/honduras/dataset.json

```diff
@@ -0,0 +1,118 @@
+{
+  "scheming_version": 1,
+  "dataset_type": "dataset",
+  "about": "Honduras CKAN dataset schema",
+  "about_url": "http://github.com/datopian/ckanext-honduras",
+  "dataset_fields": [
+    {
+      "field_name": "title",
+      "label": "Nombre del Dataset",
+      "required": true,
+      "preset": "title"
+    },
+    {
+      "field_name": "description",
+      "label": "Descripción del dataset",
+      "required": true
+    },
+    {
+      "field_name": "identifier",
+      "label": "Identificador del Dataset",
+      "required": true
+    },
+    {
+      "field_name": "theme",
+      "label": "Temática del dataset",
+      "required": true
+    },
+    {
+      "field_name": "keyword",
```

Comment:
Use the core `tags` field for this, as described [here](https://github.com/ckan/ckanext-scheming/blob/master/ckanext/scheming/ckan_dataset.json#L25) but changing the label.